docs/cicd/others-script-cd-template.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# This is just an example workflow for continuous deployment.
+# You should customize it to meet your own requirements.
+# Also you should export the following environment variables for Azure/M365 login:
+# AZURE_ACCOUNT_NAME
+# AZURE_ACCOUNT_PASSWORD
+# AZURE_SUBSCRIPTION_ID
+# AZURE_TENANT_ID
+# M365_ACCOUNT_NAME
+# M365_ACCOUNT_PASSWORD
+
+# Setup environment.
+# Sufficient permissions are required to run the commands below.
+apt install -y nodejs npm git
+
+# Checkout the code.
+git clone {RepositoryEndpoint}
```

Comment:
this doesn't look like a shell variable - are users expected to change this value? we should at least call it out in a comment or something

Reply:
Updated.